source/pastemyst/rest/paste.d

```diff
@@ -56,11 +56,11 @@ public class APIPaste : IAPIPaste
         import pastemyst.data.file : languages;
         import std.uni : toLower;
 
-        enforceHTTP(!pasties.length == 0, HTTPStatus.badRequest, "pasties arrays has to have at least one element.");
+        enforceHTTP(!pasties.length == 0, HTTPStatus.badRequest, "Pasties arrays has to have at least one element.");
```

Comment:
the general "language" style for pastemyst is to use all lowercase text, there's no particular reason for this though.